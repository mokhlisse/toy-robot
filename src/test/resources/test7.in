PLACE 2,1,EAST
MOVE
RIGHT
RIGHT
MOVE
REPORT