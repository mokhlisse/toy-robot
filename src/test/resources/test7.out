Output: 2,1,WEST